### PR TITLE
Gh 410 "Entites" changed to "entity types"

### LINKED ISSUE #410: Change "entities" for "entity types" on custom filter
From feedback on the UI, the labels on the two drop-downs (Only include these entities/edges) are confusing and don't really convey the right message. Really we're narrowing down the element group (or type). The label on the drop-downs should reflect this.

### PR DESCRIPTION
**Pull request by g609bmsma**
Fixes #410 - makes requisite changes to change label on query page from "entities" to "entity types"

### CODE REVIEW

File: ui/src/main/webapp/app/query/view-builder/view-builder.html [hunk 1]
```diff
@@ -8,42 +8,28 @@ <h3 class="md-title section-title">Add Filters</h3>
...
-                    <md-option ng-value="entity"
-                               ng-repeat="(entity, detail) in ctrl.schemaEntities | schemaGroupFilter:ctrl.entitySearchTerm">
+                    <md-option ng-value="entity" ng-repeat="(entity, detail) in ctrl.schemaEntities | schemaGroupFilter:ctrl.entitySearchTerm">
                         <div class="md-subhead">{{entity}}</div>
                         <div class="md-body-1">{{detail.description}}</div>
                     </md-option>
                 </md-select>
             </md-input-container>
             <md-input-container flex>
                 <label>Only include these edges</label>
```
Comment (d47853) [inline ui/src/main/webapp/app/query/view-builder/view-builder.html]: Could this one be as well?

Comment (g609bmsma) [reply ui/src/main/webapp/app/query/view-builder/view-builder.html]: Made that change and it doesn't seem to affect anything negatively that I can tell, so I'll add it to the changes. Just did the necessary changes to get the result beforehand, but thanks for pointing this out.

**Comment by g609bmsma** [comment]
Having changed both instances of "entity" to "entities" on that line and reloading the page after clearing the cache, it now appears to be functioning incorrectly - no longer brings up the "JunctionUse/Cardinality" options. Think changing the second "entities" back will fix this.

File: ui/src/main/webapp/app/query/view-builder/view-builder.html [hunk 2]
```diff
@@ -8,42 +8,28 @@ <h3 class="md-title section-title">Add Filters</h3>
...
         <div layout="row" layout-padding>
             <md-input-container flex>
-                <label>Only include these entities</label>
-                <md-select id="view-entities" class="no-ellipsis" multiple
-                           md-selected-text="ctrl.createViewElementsLabel(ctrl.model.viewEntities, 'entities')"
-                           ng-model="ctrl.model.viewEntities"
-                           ng-change="ctrl.onElementGroupChange('entity')"
-                           data-md-container-class="searchable-select">
+                <label>Only include these entity types</label>
+                <md-select id="view-entities" class="no-ellipsis" multiple md-selected-text="ctrl.createViewElementsLabel(ctrl.model.viewEntities, 'entity types')"
```
Comment (d47853) [inline ui/src/main/webapp/app/query/view-builder/view-builder.html]: I think this needs to stay as entities

Comment (g609bmsma) [reply ui/src/main/webapp/app/query/view-builder/view-builder.html]: The <label>entity types or the (ctrl.model.viewEntities, 'entity types')?

**Pull request closed without merging**